### PR TITLE
test-pull-request

### PR DESCRIPTION
**Pull request by EvgeniaSila**

### CODE REVIEW

**Comment by dalshekerchi** [comment]
Looks good!

**Review by dalshekerchi — APPROVED** [review]
I think this should work

**Review by hellokathylee — CHANGES_REQUESTED** [review]
Fix

**Review by hellokathylee — APPROVED** [review]

File: src/test/java/ExpensesManagerTest.java [hunk 1]
```diff
@@ -36,6 +36,7 @@
 //    }
 
 // TESTING
+//TESTING PT 2
```
Comment (dalshekerchi) [inline src/test/java/ExpensesManagerTest.java:39]: dont think this line in needed

**Review by dalshekerchi — CHANGES_REQUESTED** [review]

**Review by dalshekerchi — APPROVED** [review]

**Pull request closed without merging**